packages/yoshi-flow-editor/src/model.ts

```diff
@@ -62,6 +62,10 @@ function resolveFrom(dir: string, fileName: string): string | null {
   }
 }
 
+function shouldUseSentry():boolean {
```

Comment:
```suggestion
function shouldUseSentry(): boolean {
```
formatting police here

Reply:
Thanks!